cmd/loom/sendtx.go

```diff
@@ -50,16 +55,30 @@ func newMigrationCommand() *cobra.Command {
 			if callerChainID == "" {
 				callerChainID = cli.TxFlags.ChainID
 			}
-			return migrationTx(Id, cli.TxFlags.PrivFile, cli.TxFlags.Algo, callerChainID)
+			if Id == 2 {
+				policy := Policy{}
+				jsonData, err := ioutil.ReadFile(inputFile)
+				err = jsonpb.UnmarshalString(string(jsonData), &policy)
```

Comment:
Handle error in case you are not able to read the file.

Reply:
Done